tools/api-markdown-documenter/biome.jsonc

```diff
@@ -9,10 +9,7 @@
 			// format them conservatively without trailing commas.
 			//
 			// See also: https://github.com/microsoft/vscode/issues/102061
-			"include": [
-				// These files contain comments, but are not JSONC files
-				"**/test/test-data/**"
-			],
+			"includes": ["**/test/test-data/**", "**/src/fence.json"],
```

Comment:
Does biome merge or overwrite arrays when merging upstream configs? Does the behavior differ from extends to the implicit inheritance?